### PR TITLE
Functional tests fix

### PR DESCRIPTION
**Pull request by dlydka**

### CODE REVIEW

File: functional/register_test.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import os
+import unittest
+
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+
+
+class SubscriptionOrderTests(unittest.TestCase):
```
Comment (BognaLew) [inline functional/register_test.py]:
```suggestion
class RegisterTests(unittest.TestCase):
```

File: functional/user_login_test.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
+import os
+import unittest
+
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+
+
+class SubscriptionOrderTests(unittest.TestCase):
```
Comment (BognaLew) [inline functional/user_login_test.py]:
```suggestion
class UserLoginTests(unittest.TestCase):
```

File: src/user_dto.py [hunk 3]
```diff
@@ -28,7 +28,7 @@ class UserDto:
     email: str
     gender: str
     timestamp: str
-    id: str = str(uuid.uuid4())
+    id: str = field(default_factory=lambda: str(uuid.uuid4()))
```
Comment (BognaLew) [inline src/user_dto.py:31]: What's the reason for this change?

File: functional/register_test.py [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        self.driver.execute_script("arguments[0].click();", element)
+
+        assert self.driver.title == "Paint Drying"
+        assert self.driver.current_url == self.BASE_URL  # check if redirected
+
+    def test_register_user_exists(self):
+        self.driver.get(os.path.join(self.BASE_URL, "register"))
+
+        assert self.driver.title == 'Register - Paint Drying'
+        assert self.driver.current_url == os.path.join(self.BASE_URL, "register")  # check if redirected to login
```
Comment (BognaLew) [inline functional/register_test.py:39]: It's not necessary to check this in every test.

File: src/user_dto.py (hunk 3 above)
Comment (dlydka) [reply src/user_dto.py:31]: This change is responsible for register new user. Older one was generating uuid only once, on server start. Now it's new uuid for every user

File: functional/register_test.py (hunk 4 above)
Comment (dlydka) [reply functional/register_test.py:39]: For now I will leave this without any changes. I can reduce these checks in next sprint

**Review by MikolekN — APPROVED** [review]

**Review by BognaLew — APPROVED** [review]
